{
  "description": "sample Github_easy/o89197.json",
  "meta": {
    "full_size": 287,
    "stripped_size": 195,
    "features": [],
    "raw_features": [
      "$schema",
      "properties",
      "type",
      "type:integer",
      "type:object",
      "type:string"
    ]
  },
  "schema": {
    "$schema": "http://json-schema.org/draft-04/schema#",
    "title": "Person",
    "description": "A person from the miniapp",
    "type": "object",
    "properties": {
      "id": {
        "type": "integer",
        "description": "Unique person ID"
      },
      "first_name": {
        "type": "string"
      },
      "last_name": {
        "type": "string"
      },
      "login": {
        "type": "string"
      }
    }
  },
  "tests": [
    {
      "description": "llama 70b generated valid",
      "valid": true,
      "data": {
        "id": 1,
        "first_name": "John",
        "last_name": "Doe",
        "login": "johndoe"
      }
    },
    {
      "description": "llama-70b generated negative",
      "valid": false,
      "rust_error": "\"one\" is not of type \"integer\"",
      "python_error": "'one' is not of type 'integer'\n\nFailed validating 'type' in schema['properties']['id']:\n    {'type': 'integer', 'description': 'Unique person ID'}\n\nOn instance['id']:\n    'one'",
      "data": {
        "id": "one",
        "first_name": "John",
        "last_name": "Doe",
        "login": "johndoe"
      }
    },
    {
      "description": "llama-70b generated negative",
      "valid": false,
      "rust_error": "null is not of type \"integer\"",
      "python_error": "None is not of type 'integer'\n\nFailed validating 'type' in schema['properties']['id']:\n    {'type': 'integer', 'description': 'Unique person ID'}\n\nOn instance['id']:\n    None",
      "data": {
        "id": null,
        "first_name": "John",
        "last_name": "Doe",
        "login": "johndoe"
      }
    },
    {
      "description": "llama-70b generated negative",
      "valid": false,
      "rust_error": "1.5 is not of type \"integer\"",
      "python_error": "1.5 is not of type 'integer'\n\nFailed validating 'type' in schema['properties']['id']:\n    {'type': 'integer', 'description': 'Unique person ID'}\n\nOn instance['id']:\n    1.5",
      "data": {
        "id": 1.5,
        "first_name": "John",
        "last_name": "Doe",
        "login": "johndoe"
      }
    }
  ]
}